data/public_data/controller.py

```diff
@@ -29,19 +35,38 @@ class ETLController:
     def __init__(self):
         self.loader = Loader()
 
-    def update_movie_data(self, lower, upper, delay):
+    def run(self):
+        scheduler = BlockingScheduler()
+        movie_ids = self.loader.get_movie_id_list()
+
+        # cron for movie data
+        scheduler.add_job(self._update_movie_data, args=[1, 2000000, movie_ids])
+        scheduler.add_job(self._update_movie_data, args=[2000000, 4000000, movie_ids])
+        scheduler.add_job(self._update_movie_data, args=[4000000, 6000000, movie_ids])
+        scheduler.add_job(self._update_movie_data, args=[6000000, 8000000, movie_ids])
```

Comment:
What happens after all finished? Is there a way to track new movie when this finishes?